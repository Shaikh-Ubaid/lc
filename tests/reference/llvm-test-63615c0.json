{
    "basename": "llvm-test-63615c0",
    "cmd": "lc --no-color --show-llvm {infile} -o {outfile}",
    "infile": "tests/test.cpp",
    "infile_hash": "9e5ef4c37b7e9e1f13f0082ce65459c1e596430887537eb8fd656b77",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "llvm-test-63615c0.stdout",
    "stdout_hash": "693feb9b391bb484b8222d405bef48b74ca2ab345a8bb0bfcf1f78f5",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}