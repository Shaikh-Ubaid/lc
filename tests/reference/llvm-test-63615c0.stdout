; ModuleID = 'LFortran'
source_filename = "LFortran"

define i32 @f(i32* %x) {
.entry:
  %__return_var = alloca i32, align 4
  %result = alloca i32, align 4
  %0 = load i32, i32* %x, align 4
  %1 = sdiv i32 %0, 42
  store i32 %1, i32* %result, align 4
  %2 = load i32, i32* %result, align 4
  store i32 %2, i32* %__return_var, align 4
  br label %return

unreachable_after_return:                         ; No predecessors!
  br label %return

return:                                           ; preds = %unreachable_after_return, %.entry
  %3 = load i32, i32* %__return_var, align 4
  ret i32 %3
}
